### PR TITLE
4448 homelessness support line

### PR DESCRIPTION
**Pull request by phiden**
adding homelessness support line box to housing assistance, careers, healthcare. See https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4448

### CODE REVIEW

**Comment by phiden** [comment]
Please don't merge without @peggygannon's ok!

File: content/pages/employment/index.md
Comment (robertfairhead) [inline content/pages/employment/index.md]: Is there any particular reason you added the event listener down here instead of inline?

```
<a id="crisis-expander-link" onClick="document.getElementById('crisis-expander-content').classList.toggle('expander-content-closed')">
```

I'm not sure how other modals are handled, but I'd suggest we try to be consistent between inlining the function, a `<script>` tag in the body, and offloading it into a common JS snippet.

Nothing particularly wrong with this, just my ignorance of how it's handled elsewhere.

**Review by robertfairhead — APPROVED** [review]
One question but not a blocker. 👍

File: content/pages/employment/index.md
Comment (phiden) [reply content/pages/employment/index.md]: Inline is literally how it's handled everywhere, so we're definitely being consistent, but I can foresee this becoming an issue in the future. It's on my list of things that will need to be standardized!

**Review by peggygannon — CHANGES_REQUESTED** [review]
Hi, @phiden A few changes: 
on the Employment page: 
1. Please delete the health care intro paragraph on that page the one that starts... With VA health care, you're covered for regular checkups....
2. please add a period after Contact a CEC (don't hyperlink the period)

**Comment by phiden** [comment]
Ok @peggygannon changes made!

**Review by peggygannon — COMMENTED** [review]
LGTM!

**Review by peggygannon — APPROVED** [review]

**Comment by bethpotts** [comment]
@phiden @peggygannon This wasn't supposed to be merged yet! Can we pull it out of today's build still? We have to get stakeholder sign-off first.

**Pull request merged**
